R/plot_jags_tracedx.R

```diff
@@ -60,19 +60,23 @@ plot_jags_trace <- function(data,
         # Changing parameter name to reflect the input
         dplyr::mutate(Parameter = paste0("iso = ", j, ", parameter = ",
                                          .data$Parameter_sub, ", strat = ",
-                                         i),
-                      value = log(.data$value))
+                                         i))
       # Assigning attributes, which are needed to run ggs_density
       attributes(visualize_jags_plot) <- c(attributes(visualize_jags_plot),
                                            attributes_jags)
       # Creating density plot
       traceplot <- ggmcmc::ggs_traceplot(visualize_jags_plot) +
         ggplot2::theme_bw() +
-        ggplot2::labs(x = "log(value)") +
-        ggplot2::theme(legend.position = "bottom")
+        ggplot2::labs(x = "iterations", y = "log(value)") +
+        ggplot2::theme(legend.position = "bottom") +
+        ggplot2::scale_y_log10()
```

Comment:
Try this:

```suggestion
        ggplot2::scale_y_log10(labels = scales::label_comma())
```

It should improve the formatting of the y-axis tick labels for the `alpha` parameter